app/services/to_cocina/work/mapper.rb

```diff
@@ -55,15 +55,12 @@ def object_type
       end
 
       def document?
-        return false if content.content_files.empty?
-
-        # All files are PDF or hidden
-        return false unless content.content_files.all? do |content_file|
-          content_file.hidden? || content_file.pdf?
-        end
+        # All shown (not hidden) files must be PDFs to be considered a document type
+        shown_files.any? && shown_files.all?(&:pdf?)
+      end
 
-        # There is at least one non-hidden PDF
-        content.content_files.any? { |content_file| !content_file.hidden? && content_file.pdf? }
+      def shown_files
+        @shown_files ||= content.content_files.reject(&:hidden?)
```

Comment:
Scope on content?

Reply:
I like that idea. I changed this to a scope on ContentFile and a helper method on Content. Have a look-see.